.pre-commit-config.yaml

```diff
@@ -57,9 +57,14 @@ repos:
   - id: go-critic
     args: [-disable, "#experimental,sloppyTypeAssert"]
 - repo: https://github.com/ansible/ansible-lint.git
-  rev: v5.4.0
+  rev: v6.8.2
   hooks:
   - id: ansible-lint
+    always_run: false
+    exclude: '^(docs/|examples/|community/examples/)'
+    types: [yaml]
```

Comment:
I am assuming (as proven by the files in which errors were caught) that this will also pick up files with `.yml` suffix.

Reply:
if you follow the chain starting at [the pre-commit docs](https://pre-commit.com/#filtering-files-with-types) you find this [section of code](https://github.com/pre-commit/identify/blob/56aa6afbe66c9da96dcdb66d9a61632f45c6b44b/identify/extensions.py#L253-L256) that demonstrates both `yml` and `yaml` match the `yaml` type.